### PR TITLE
mist: update repo name

### PR DESCRIPTION
**Pull request by ninxsoft**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by ninxsoft** [comment]
* The `mist` GitHub repository has been renamed to [mist-cli](https://github.com/ninxsoft/mist-cli).
* The formula description has been updated.
* The formula `shasum` had an incorrect / outdated value - this is now fixed.

File: Formula/mist.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Mist < Formula
-  desc "Mac command-line tool that automatically downloads macOS Installers / Firmwares"
-  homepage "https://github.com/ninxsoft/Mist"
-  url "https://github.com/ninxsoft/Mist/archive/refs/tags/v1.8.tar.gz"
-  sha256 "c3a55d018f384090e5bccf625fe2c9720102218fe9126d651c4583cc504ae25c"
+  desc "Mac command-line tool that automatically downloads macOS Firmwares / Installers"
```
Comment (carlocab) [inline Formula/mist.rb:2]: Is changing this necessary?

**Comment by carlocab** [comment]
For future reference:

Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request merged**